--- nqptp-1.2.4/debian/control.org	2023-10-20 15:53:08.013031922 +0200
+++ nqptp-1.2.4/debian/control	2023-10-20 15:50:02.656036622 +0200
@@ -1,19 +1,22 @@
 Source: nqptp
-Section: unknown
+Section: libs
 Priority: optional
 Maintainer: Bitkeeper <bitkeeper@users.noreply.github.com>
 Rules-Requires-Root: no
 Build-Depends:
  debhelper-compat (= 13),
 Standards-Version: 4.7.2
-Homepage: <insert the upstream URL, if relevant>
-#Vcs-Browser: https://salsa.debian.org/debian/nqptp
-#Vcs-Git: https://salsa.debian.org/debian/nqptp.git
+Homepage: https://github.com/mikebrady/nqptp
+Vcs-Browser: https://github.com/mikebrady/nqptp
+Vcs-Git: https://github.com/mikebrady/nqptp
+

 Package: nqptp
 Architecture: any
 Depends:
  ${shlibs:Depends},
  ${misc:Depends},
-Description: <insert up to 60 chars description>
- <Insert long description, indented with spaces.>
+Description: nqptp is a daemon that monitors timing data from any PTP clocks – up to 64 – it sees on ports 319 and 320.
+ nqptp is a daemon that monitors timing data from any PTP clocks – up to 64 – it sees on ports 319 and 320.
+ It maintains records for each clock, identified by Clock ID and IP.
+ It is a companion application to Shairport Sync and provides timing information for AirPlay 2 operation.
