--- alsacap-1.0.1/debian/control.org	2023-10-17 10:53:34.290999345 +0200
+++ alsacap-1.0.1/debian/control	2023-10-17 10:55:07.691454011 +0200
@@ -1,19 +1,18 @@
 Source: alsacap
-Section: unknown
+Section: sound
 Priority: optional
-Maintainer: Bitkeeper <bitkeeper@users.noreply.github.com>
+Maintainer: bitkeeper <bitkeeper@users.noreply.github.com>
 Rules-Requires-Root: no
 Build-Depends:
  debhelper-compat (= 13),
 Standards-Version: 4.6.2
-Homepage: <insert the upstream URL, if relevant>
-#Vcs-Browser: https://salsa.debian.org/debian/alsacap
-#Vcs-Git: https://salsa.debian.org/debian/alsacap.git
+Homepage: https://github.com/bubbapizza/alsacap
+Vcs-Browser: https://github.com/bitkeeper/alsacap
+Vcs-Git: https://github.com/bitkeeper/alsacap.git

 Package: alsacap
 Architecture: any
-Depends:
- ${shlibs:Depends},
- ${misc:Depends},
-Description: <insert up to 60 chars description>
- <Insert long description, indented with spaces.>
+Depends: ${shlibs:Depends}, ${misc:Depends}, libasound2
+Description: Alsacap was originally
+ written by Volker Schatz and it shows the capabilities of alsa device.
+ Shawn Wilson added autoconfigure to the project.
\ No newline at end of file
