--- squeezelite.org/debian/rules	2022-01-31 19:18:37.311123175 +0100
+++ ./squeezelite/debian/rules	2022-01-31 19:19:01.150701900 +0100
@@ -6,7 +6,7 @@

 # Enable ffmpeg, soxr resampling, visualisation export and disable dlopening
 # libraries
-export OPTS := -DDSD -DFFMPEG -DRESAMPLE -DVISEXPORT -DLINKALL -DIR -DUSE_SSL
+export OPTS := -DDSD -DFFMPEG -DRESAMPLE -DVISEXPORT -DLINKALL -DIR -DUSE_SSL -DGPIO -DRPI

 # Enable all hardening build flags
 export DEB_BUILD_MAINT_OPTIONS := hardening=+all
