--- upmpdcli-1.5.12/debian/control	2021-12-07 20:29:55.542486340 +0000
+++ ../../../build/upmpdcli-1.5.12/debian/control	2021-11-15 20:19:28.532731546 +0000
@@ -4,11 +4,10 @@
 Maintainer: Jean-Francois Dockes <jf@dockes.org>
 # note: libexpat1-dev and libcurl-dev are only here because of pkg-config:
 # not directly needed for building upmpdcli
-Build-Depends: debhelper (>= 9.0.0), dh-systemd, autotools-dev,
+Build-Depends: debhelper (>= 9.0.0), autotools-dev,
                dh-autoreconf, pkg-config,
                libmpdclient-dev, libmicrohttpd-dev, libjsoncpp-dev,
                libexpat1-dev, libcurl4-gnutls-dev,
-               qt5-default, qt5-qmake, qtbase5-dev,
                libupnpp6-dev (>= 0.20.0)
 Standards-Version: 3.9.8
 Homepage: http://www.lesbonscomptes.com/upmpdcli
@@ -20,17 +19,17 @@
 Description: UPnP Media Renderer front-end to MPD, the Music Player Daemon
  upmpdcli acts as an UPnP Media Renderer on the front-end so that you can
  control it with any UPnP controller like, e.g. Audionet or BubbleUPnP on an
- Android tablet. It uses an MPD instance to actually play the tracks. 
+ Android tablet. It uses an MPD instance to actually play the tracks.
  A typical configuration might have for example, MPD running on a Raspberry
  PI, with upmpdcli on the same host or any other Linux PC on the network.
 
 
-Package: upmpdcli-config
-Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
-Section: contrib/sound
-Description: GUI configuration editor for upmpdcli.
- upmpdcli-config is a qt-based GUI for editing an upmpdcli configuration file.
+# Package: upmpdcli-config
+# Architecture: any
+# Depends: ${shlibs:Depends}, ${misc:Depends}
+# Section: contrib/sound
+# Description: GUI configuration editor for upmpdcli.
+#  upmpdcli-config is a qt-based GUI for editing an upmpdcli configuration file.
 
 Package: scweb
 Architecture: all
