--- ashuffle-3.12.5/debian/control	2021-12-07 17:22:17.383244716 +0000
+++ ../../../build/ashuffle_3.12.5/debian/control	2021-12-07 17:31:08.137883200 +0000
@@ -1,25 +1,19 @@
 Source: ashuffle
+Section: sound
 Priority: optional
 Maintainer: Bitkeeper <bitkeeper@users.noreply.github.com>
-Build-Depends: debhelper-compat (= 13)
+Build-Depends: debhelper-compat (= 13),
+               libmpdclient-dev,
+               meson,
+               cmake
 Standards-Version: 4.5.1
-Section: libs
-Homepage: <insert the upstream URL, if relevant>
-#Vcs-Browser: https://salsa.debian.org/debian/ashuffle
-#Vcs-Git: https://salsa.debian.org/debian/ashuffle.git
+Homepage: https://github.com/joshkunz/ashuffle
+Vcs-Browser: https://github.com/joshkunz/ashuffle
+Vcs-Git: https://github.com/joshkunz/ashuffle.git
 Rules-Requires-Root: no
 
-Package: ashuffle-dev
-Section: libdevel
+Package: ashuffle
 Architecture: any
-Multi-Arch: same
-Depends: ashuffleBROKEN (= ${binary:Version}), ${misc:Depends}
-Description: <insert up to 60 chars description>
- <insert long description, indented with spaces>
-
-Package: ashuffleBROKEN
-Architecture: any
-Multi-Arch: same
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: <insert up to 60 chars description>
- <insert long description, indented with spaces>
+Description: ashuffle is an application for automatically shuffling your MPD library in a similar way to a more standard music player's "shuffle library" feature.
+ ashuffle works like any other MPD client, and can be used alongside your other MPD frontends.
