--- ashuffle-3.14.1/debian/control.org	2023-10-22 14:07:41.216179190 +0200
+++ ashuffle-3.14.1/debian/control	2023-10-22 14:09:32.993351641 +0200
@@ -1,19 +1,22 @@
 Source: ashuffle
-Section: unknown
+Section: sound
 Priority: optional
 Maintainer: Bitkeeper <bitkeeper@users.noreply.github.com>
 Rules-Requires-Root: no
 Build-Depends:
  debhelper-compat (= 13),
+ libmpdclient-dev,
+ meson,
+ cmake
 Standards-Version: 4.7.2
-Homepage: <insert the upstream URL, if relevant>
-#Vcs-Browser: https://salsa.debian.org/debian/ashuffle
-#Vcs-Git: https://salsa.debian.org/debian/ashuffle.git
+Homepage: https://github.com/joshkunz/ashuffle
+Vcs-Browser: https://github.com/joshkunz/ashuffle
+Vcs-Git: https://github.com/joshkunz/ashuffle.git

 Package: ashuffle
 Architecture: any
 Depends:
  ${shlibs:Depends},
  ${misc:Depends},
-Description: <insert up to 60 chars description>
- <Insert long description, indented with spaces.>
+Description: ashuffle is an application for automatically shuffling your MPD library in a similar way to a more standard music player's "shuffle library" feature.
+ ashuffle works like any other MPD client, and can be used alongside your other MPD frontends.
