--- ashuffle-3.12.5/debian/rules	2021-12-07 17:22:17.378244805 +0000
+++ ../../../build/ashuffle_3.12.5/debian/rules	2021-11-22 20:31:28.026333255 +0000
@@ -22,6 +22,8 @@

 # dh_make generated override targets.
 # This is an example for Cmake (see <https://bugs.debian.org/641051>).
-#override_dh_auto_configure:
-#	dh_auto_configure -- \
-#	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
+override_dh_auto_configure:
+	git submodule update --init --recursive
+	meson -Dbuildtype=release build
+	dh_auto_configure --buildsystem=meson --
+#	-DCMAKE_LIBRARY_PATH=$(DEB_HOST_MULTIARCH)
