--- mpd-0.24/debian/rules.org	2025-03-11 21:50:18.539473886 +0100
+++ mpd-0.24/debian/rules	2025-03-11 21:50:52.216776952 +0100
@@ -31,39 +31,42 @@
 		-Dzzip=enabled

 # Autodiscovery support
-ENABLE_AUTODISCOVERY=-Dneighbor=true \
-		-Dupnp=pupnp \
+ENABLE_AUTODISCOVERY=-Dneighbor=false \
+		-Dupnp=disabled \
 		-Dzeroconf=avahi

 # Client support
-ENABLE_CLIENTS=-Dipv6=enabled \
+ENABLE_CLIENTS=-Dipv6=disabled \
 	 	-Dtcp=true \
 	 	-Dlocal_socket=true

 # Storage support
-ENABLE_STORAGE=-Dnfs=enabled \
+ENABLE_STORAGE=-Dnfs=disabled \
 	 	${ENABLE_SMBCLIENT} \
-	 	-Dwebdav=enabled
+	 	-Dsmbclient=disabled \
+		-Dudisks=disabled \
+	 	-Dwebdav=disabled

 # File format support
 ENABLE_FILEFORMATS=-Dfaad=enabled \
 		-Dadplug=enabled \
 		-Daudiofile=disabled \
+		-Ddsd=true \
 		-Dffmpeg=enabled \
 		-Dflac=enabled \
-		-Dfluidsynth=enabled \
-		-Dgme=enabled \
+		-Dfluidsynth=disabled \
+		-Dgme=disabled \
 		-Dmad=enabled \
-		-Dmikmod=enabled \
-		-Dmodplug=enabled \
-		-Dmpcdec=enabled \
+		-Dmikmod=disabled \
+		-Dmodplug=disabled \
+		-Dmpcdec=disabled \
 		-Dmpg123=enabled \
-		-Dopenmpt=enabled \
+		-Dopenmpt=disabled \
 		-Dopus=enabled \
-		-Dsidplay=enabled \
-		-Dsndfile=enabled \
+		-Dsidplay=disabled \
+		-Dsndfile=disabled \
 		-Dwavpack=enabled \
-		-Dwildmidi=enabled
+		-Dwildmidi=disabled

 # Other features
 ENABLE_OTHER=-Dcue=true \
@@ -73,7 +76,7 @@
 		-Dlibmpdclient=enabled \
 		-Dinotify=true \
 		-Dsoxr=enabled \
-		-Dlibsamplerate=enabled \
+		-Dlibsamplerate=disabled \
 		-Dsqlite=enabled \
 		-Dsyslog=enabled \
 		-Dyajl=enabled \
@@ -81,23 +84,23 @@

 # Metadata support
 ENABLE_METADATA=-Did3tag=enabled \
-		-Dchromaprint=enabled
+		-Dchromaprint=disabled

 # Playback support
 # Note: ALSA is in WITH_LINUX
 ENABLE_PLAYBACKS=-Dfifo=true \
-		-Dsndio=enabled \
+		-Dsndio=disabled \
 		-Drecorder=true \
 		-Dhttpd=true \
-		-Djack=enabled \
-		-Dao=enabled \
-		-Doss=enabled \
-		-Dopenal=enabled \
+		-Djack=disabled \
+		-Dao=disabled \
+		-Doss=disabled \
+		-Dopenal=disabled \
 		-Dpipe=true \
-		-Dpipewire=enabled \
-		-Dpulse=enabled \
+		-Dpipewire=disabled \
+		-Dpulse=disabled \
 		-Dshout=enabled \
-		-Dsnapcast=true
+		-Dsnapcast=false

 # Streaming encoder support
 # Note: FLAC and Opus enabled under file formats, vorbisenc at top (!tremor)
@@ -108,12 +111,12 @@
 # Streaming support
 ENABLE_STREAMING=-Dcdio_paranoia=enabled \
 		-Dcurl=enabled \
-		-Dsoundcloud=enabled \
+		-Dsoundcloud=disabled \
 		-Dqobuz=disabled \
-		-Dmms=enabled
+		-Dmms=disabled

 # Sphinx documentation
-ENABLE_DOCUMENTATION=-Ddocumentation=enabled
+ENABLE_DOCUMENTATION=-Ddocumentation=disabled

 ###############################################################

