--- mpd-0.23.5/debian/rules.org	2022-01-05 12:13:56.836000000 +0000
+++ mpd-0.23.5/debian/rules	2022-01-05 12:11:38.782337102 +0000
@@ -32,7 +32,7 @@
 
 # Autodiscovery support
 ENABLE_AUTODISCOVERY=-Dneighbor=true \
-		-Dupnp=pupnp \
+		-Dupnp=disabled \
 		-Dzeroconf=avahi
 
 # Client support
@@ -49,10 +49,11 @@
 ENABLE_FILEFORMATS=-Dfaad=enabled \
 		-Dadplug=enabled \
 		-Daudiofile=enabled \
+		-Ddsd=true \
 		-Dffmpeg=enabled \
 		-Dflac=enabled \
-		-Dfluidsynth=enabled \
-		-Dgme=enabled \
+		-Dfluidsynth=disabled \
+		-Dgme=disabled \
 		-Dmad=enabled \
 		-Dmikmod=enabled \
 		-Dmodplug=enabled \
@@ -60,10 +61,10 @@
 		-Dmpg123=enabled \
 		-Dopenmpt=enabled \
 		-Dopus=enabled \
-		-Dsidplay=enabled \
-		-Dsndfile=enabled \
+		-Dsidplay=disabled \
+		-Dsndfile=disabled \
 		-Dwavpack=enabled \
-		-Dwildmidi=enabled
+		-Dwildmidi=disabled
 
 # Other features
 ENABLE_OTHER=-Dcue=true \
@@ -73,7 +74,7 @@
 		-Dlibmpdclient=enabled \
 		-Dinotify=true \
 		-Dsoxr=enabled \
-		-Dlibsamplerate=enabled \
+		-Dlibsamplerate=disabled \
 		-Dsqlite=enabled \
 		-Dsyslog=enabled \
 		-Dyajl=enabled \
@@ -89,13 +90,13 @@
 		-Dsndio=enabled \
 		-Drecorder=true \
 		-Dhttpd=true \
-		-Djack=enabled \
-		-Dao=enabled \
-		-Doss=enabled \
+		-Djack=disabled \
+		-Dao=disabled \
+		-Doss=disabled \
 		-Dopenal=enabled \
 		-Dpipe=true \
 		-Dpipewire=enabled \
-		-Dpulse=enabled \
+		-Dpulse=disabled \
 		-Dshout=enabled \
 		-Dsnapcast=true
 
@@ -108,7 +109,7 @@
 # Streaming support
 ENABLE_STREAMING=-Dcdio_paranoia=enabled \
 		-Dcurl=enabled \
-		-Dsoundcloud=enabled \
+		-Dsoundcloud=disabled \
 		-Dqobuz=disabled \
 		-Dmms=enabled
 
