diff --git a/src/guiconfig.ts b/src/guiconfig.ts
index 90b5b2f..a368297 100644
--- a/src/guiconfig.ts
+++ b/src/guiconfig.ts
@@ -4,6 +4,7 @@ export interface GuiConfig {
     hide_capture_device: boolean
     hide_playback_device: boolean
     hide_rate_monitoring: boolean
+    hide_file_manager: boolean
     coeff_dir: string
 }

@@ -21,6 +22,7 @@ export function defaultGuiConfig(): GuiConfig {
         hide_capture_device: false,
         hide_playback_device: false,
         hide_rate_monitoring: false,
+        hide_file_manager: false,
         coeff_dir: '',
     }
 }
\ No newline at end of file
diff --git a/src/index.tsx b/src/index.tsx
index 45f51e0..0a43b29 100644
--- a/src/index.tsx
+++ b/src/index.tsx
@@ -259,7 +259,8 @@ class CamillaConfig extends React.Component<
           <Tab>Filters {errors({path: ['filters'], includeChildren: true}) && <ErrorIcon/>}</Tab>
           <Tab>Mixers {errors({path: ['mixers'], includeChildren: true}) && <ErrorIcon/>}</Tab>
           <Tab>Pipeline {errors({path: ['pipeline'], includeChildren: true}) && <ErrorIcon/>}</Tab>
-          <Tab>Files</Tab>
+          {this.state.guiConfig.hide_file_manager===false &&
+          <Tab>Files</Tab>}
           <Tab>Shortcuts</Tab>
         </TabList>
         <TabPanel/>
@@ -328,14 +328,15 @@ class CamillaConfig extends React.Component<
           />
         </TabPanel>
         <TabPanel>
+          {this.state.guiConfig.hide_file_manager===false &&
           <Files
               currentConfigFile={currentConfigFile}
               config={config}
               setCurrentConfig={this.setCurrentConfig}
               setCurrentConfigFileName={this.setCurrentConfigFileName}
               updateConfig={this.updateConfig}
               saveNotify={this.saveNotify}
               guiConfig={this.state.guiConfig}
-          />
+          />}
         </TabPanel>
         <TabPanel>
           <Shortcuts
