diff --git a/src/guiconfig.ts b/src/guiconfig.ts
index 00b8bf8..c22af65 100644
--- a/src/guiconfig.ts
+++ b/src/guiconfig.ts
@@ -5,6 +5,7 @@ export interface GuiConfig {
   hide_playback_device: boolean
   hide_rate_monitoring: boolean
   hide_multithreading: boolean
+  hide_file_manager: boolean
   coeff_dir: string
   supported_capture_types?: CaptureType[]
   supported_playback_types?: PlaybackType[]
@@ -63,6 +64,7 @@ export function defaultGuiConfig(): GuiConfig {
     hide_playback_device: false,
     hide_rate_monitoring: false,
     hide_multithreading: false,
+    hide_file_manager: false,
     coeff_dir: "",
     apply_config_automatically: false,
     save_config_automatically: false,
diff --git a/src/index.tsx b/src/index.tsx
index b96f6ae..c35789b 100644
--- a/src/index.tsx
+++ b/src/index.tsx
@@ -344,7 +344,8 @@ class CamillaConfig extends React.Component<
             <Tab>Mixers {errors.hasErrorsFor("mixers") && <ErrorIcon />}</Tab>
             <Tab>Processors {errors.hasErrorsFor("processors") && <ErrorIcon />}</Tab>
             <Tab>Pipeline {errors.hasErrorsFor("pipeline") && <ErrorIcon />}</Tab>
-            <Tab>Files</Tab>
+            {this.state.guiConfig.hide_file_manager===false &&
+            <Tab>Files</Tab>}
             <Tab>Shortcuts</Tab>
           </TabList>
           <TabPanel />
@@ -379,6 +380,7 @@ class CamillaConfig extends React.Component<
             <PipelineTab config={config} updateConfig={this.updateConfig} errors={errors.forSubpath("pipeline")} />
           </TabPanel>
           <TabPanel>
+            {this.state.guiConfig.hide_file_manager===false &&
             <Files
               currentConfigFile={currentConfigFile}
               config={config}
@@ -388,6 +390,6 @@ class CamillaConfig extends React.Component<
               saveNotify={this.saveNotify}
               guiConfig={this.state.guiConfig}
-            />
+            />}
           </TabPanel>
           <TabPanel>
             <Shortcuts
