diff --git a/src/compactview.tsx b/src/compactview.tsx
index 2e63522..8122fdf 100644
--- a/src/compactview.tsx
+++ b/src/compactview.tsx
@@ -40,7 +40,6 @@ export function CompactView(props: {
     <DisableCompactViewButton disableCompactView={disableCompactView}/>
     <VolumeBox vuMeterStatus={defaultStatus()} setMessage={() => {}}/>
     <ShortcutSections sections={guiConfig.custom_shortcuts} config={config} updateConfig={updateConfig}/>
-    <QuickConfigSwitch setConfig={setConfig} currentConfigName={currentConfigName}/>
   </div>
 }
 
