--- alsa-cdsp-1.0.0/debian/rules.org	2022-01-23 20:31:35.226453334 +0100
+++ alsa-cdsp-1.0.0/debian/rules	2022-01-23 20:31:42.468336843 +0100
@@ -13,6 +13,9 @@
 # package maintainers to append LDFLAGS
 #export DEB_LDFLAGS_MAINT_APPEND = -Wl,--as-needed

+export DEB_CFLAGS_STRIP = -fstack-protector-strong -Wformat -Werror=format-security
+export DEB_LDFLAGS_STRIP = -Wl,-z,relro -Wl,-z,now -Wl,--as-needed
+

 %:
 	dh $@
