--- camilladsp-0.6.3/Cargo.toml.org	2022-04-10 12:01:05.380529460 +0200
+++ camilladsp-0.6.3/Cargo.toml	2022-04-10 12:22:20.474806161 +0200
@@ -90,3 +92,13 @@

 [profile.release]
 codegen-units = 1
+
+[package.metadata.deb]
+maintainer-scripts = "debian/"
+#license-file = ["LICENSE.txt", "0"]
+assets = [
+    ["target/release/camilladsp", "usr/local/bin/", "755"],
+    ["README.md", "usr/share/doc/camilladsp/README.md", "644"],
+]
+systemd-units = { enable = false, start = false }
+changelog = "CHANGELOG.md"
