--- camilladsp-0.6.3/Cargo.toml.org	2022-04-10 12:01:05.380529460 +0200
+++ camilladsp-0.6.3/Cargo.toml	2022-04-10 12:22:20.474806161 +0200
@@ -5,6 +5,8 @@
 edition = "2021"
 description = "A flexible tool for processing audio"
 rust-version = "1.61"
+readme = "README.md"
+license-file = "LICENSE.txt"

 [features]
 default = ["websocket"]
@@ -84,3 +86,13 @@
 [[bench]]
 name = "filters"
 harness = false
+
+[package.metadata.deb]
+maintainer-scripts = "debian/"
+#license-file = ["LICENSE.txt", "0"]
+assets = [
+    ["target/release/camilladsp", "usr/local/bin/", "755"],
+    ["README.md", "usr/share/doc/camilladsp/README.md", "644"],
+]
+systemd-units = { enable = false, start = false }
+changelog = "CHANGELOG.md"
