--- shairport-sync-dev.org/debian/rules.org	2026-07-28 18:49:47.010007194 -0400
+++ shairport-sync-dev/debian/rules	2026-07-28 18:55:46.496571889 -0400
@@ -23,7 +23,7 @@
 		--with-pipe \
 		--without-configfiles \
 		--with-piddir=/run/shairport-sync \
-		--with-libdaemon \
+		--without-libdaemon \
 		--with-ssl=openssl \
 		--with-soxr \
 		--with-metadata \
@@ -31,10 +31,10 @@
 		--with-metadata-multicast \
 		--with-avahi \
 		--with-alsa \
-		--with-jack \
-		--with-pulseaudio \
-		--with-pipewire \
-		--with-convolution \
+		--without-jack \
+		--without-pulseaudio \
+		--without-pipewire \
+		--without-convolution \
 		--with-dbus-interface \
 		--with-mpris-interface \
 		--with-mqtt-client \
